bucket/springboot.json

```diff
@@ -0,0 +1,23 @@
+{
+    "homepage": "https://projects.spring.io/spring-boot/",
+    "version": "2.0.0.RELEASE",
+    "license": "Apache 2.0",
+    "hash": "45362FCB133E5DB84A771DD37AF4D8B9655D936877F8E537DBDE2C1D648E488A",
```

Comment:
The has is available at `$url.sha256`. Add the following to autoupdate to update the hash without needing to download the file: 

```json
"hash": {
    "mode": "extract",
    "url": "$url.sha256"
}
```

Reply:
The `mode` property can be omitted, it defaults to `extract`